### PR TITLE
Create bundle+index for 16.2 and 17.0 in github actions

### PR DESCRIPTION
**Pull request by stuggi**

### CODE REVIEW

**Review by abays — COMMENTED** [review]
+1 from me, but should probably let @dprince give the final approval

**Review by dprince — APPROVED** [review]

File: .github/create_bundle.sh [hunk 1]
```diff
@@ -9,6 +9,7 @@ echo "${GITHUB_SHA}"
 echo "${BASE_IMAGE}"
 echo "${AGENT_IMAGE}"
 echo "${DOWNLOADER_IMAGE}"
+echo "${OSP_RELEASE}"
```
[the remark this answers is not in the document]
Comment (stuggi) [reply .github/create_bundle.sh:12]: gets set as part of the  menager_default_images for the release https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/config/default/manager_default_images_17.0.yaml#L30

**Review by olliewalsh — APPROVED** [review]
/lgtm

**Pull request merged**